src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__startDay--sun calendar__monthLength--31">
```

Comment:
The use of BEM naming convention is incorrect. The class `calendar__startDay--sun` and `calendar__monthLength--31` seems to be a modifier but it's used as if it's an element. Remember, in BEM, modifiers are used to denote a different version of a block or an element, and they are separated by double hyphens (--). An element is a part of a block and it's separated by double underscores (__). You might want to rename your classes to follow the BEM convention correctly.